go/vt/vttest/local_cluster.go

```diff
@@ -533,7 +533,7 @@ func (db *LocalCluster) createDatabases() error {
 			sql = append(sql, fmt.Sprintf("create database `%s`", dbname))
 		}
 	}
-	return db.Execute(sql, "")
+	return db.ExecuteWithSuperReadOnly(sql, "")
```

Comment:
Aren't we simply avoiding really testing the new functionality whenever we use this?